packages/mobile/src/screens/collection-screen/useFetchCollectionLineup.ts

```diff
@@ -24,17 +21,13 @@ import { getOfflineTrackIds } from 'app/store/offline-downloads/selectors'
 
 const { getCollection } = cacheCollectionsSelectors
 const { getCollectionTracksLineup } = collectionPageSelectors
-const { makeGetTableMetadatas } = lineupSelectors
 const { getPositions } = queueSelectors
-const { getIsReachable } = reachabilitySelectors
-
-const getTracksLineup = makeGetTableMetadatas(getCollectionTracksLineup)
 
 /**
  * Returns a collection lineup, supports boths online and offline
  * @param collectionId the numeric collection id
  */
-export const useCollectionLineup = (
+export const useFetchCollectionLineup = (
```

Comment:
This is kinda strange because it no longer follows the same pattern as `useFavoritesLineup`. Could the new selectors be incorporated into `useCollectionLineup`?

Reply:
i think it def can! but then it feels a bit weird that the hook just magically knows what we need? thoughts?